apps/ex_nvr_web/test/ex_nvr_web/live/device_live_test.exs

```diff
@@ -153,6 +154,34 @@ defmodule ExNVRWeb.DeviceLiveTest do
       assert result =~ "invalid rtsp uri"
       assert Enum.empty?(Devices.list())
     end
+
+    test "renders errors on wrong vendor", %{conn: conn} do
```

Comment:
No need to add this test. Add only tests on `devices_test.exs` (test that the values are stored and updated)